src/pages/set-location/components/SelectedChipsSheet.tsx

```
@@ -0,0 +1,30 @@
+  return (
+    <div className='flex flex-col gap-[1rem] rounded-t-[1rem] px-[2rem] py-[1.1rem] bg-white shadow-[0px_-4px_10px_0px_rgba(0,0,0,0.04)]'>
+      <span className='title-sb-12'>
+        <span className='text-primary-700'>{selectedLocationsId.size} </span>
+        <span className='text-black'>/ {MAX_SELECTED}</span>
+      </span>
+      <div className='flex flex-wrap gap-[0.8rem]'>
+        {[...selectedLocationsId].map(chip => (
+          <SelectChip
+            title={chip.name}
+            handleDeleteChip={() => handleDeleteLocation(chip)}
+          />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)